epictrack-web/src/components/work/workStaff/WorkStaffList.tsx

```diff
@@ -165,12 +196,18 @@ const WorkStaffList = () => {
         header: "Responsible EPD",
         filterVariant: "multi-select",
         filterSelectOptions: responsibleEpdFilter,
+        Cell: ({ renderedCellValue }) => {
+          return renderedCellValue;

```

Comment:
what is this doing? is it not redundant?

Reply:
removed it